client/web/compose/src/views/Public/Pages/Records/View.vue

```diff
@@ -245,6 +274,23 @@ export default {
         this.$router.push({ name: 'page.record.edit', params: this.$route.params })
       }
     },
+
+    handleRedirectToPrevOrNext (index) {
+      const recordID = this.recordIDs[index]
+
+      if (recordID !== undefined) {
+        const route = {
+          name: 'page.record',
+          params: {
+            pageID: this.page.pageID,
+            recordID,
+          },
+          query: null,
+        }
+
+        this.$router.push(route)
```

Comment:
It will probably have to be handled differently than this. Since you can have this open in a modal aswell. You'll need to come up with a solution, or atleast think about it and we can resolve it together if you're not sure.